bin/tigmint-make

```diff
@@ -149,11 +149,36 @@ tigmint: \
 	$(draft).$(reads).as$(as).nm$(nm).molecule.size$(minsize).trim$(trim).window$(window).span$(span).breaktigs.fa \
 	$(draft).tigmint.fa
 
+# Index the linked reads 
+tigmint-index: $(draft).fa.bwt
+
+# Align the linked reads (using bwa alignment) 
+tigmint-align: $(draft).$(reads).sortbx.bam
+
+# Perform the tigmint-molecule step for linked reads 
+tigmint-molecule: $(draft).$(reads).as$(as).nm$(nm).molecule.size$(minsize).bed
+
+# Perform the final tigmint-cut step for linked reads
+tigmint-cut: \
+        $(draft).$(reads).as$(as).nm$(nm).molecule.size$(minsize).trim$(trim).window$(window).span$(span).breaktigs.fa \
+        $(draft).tigmint.fa
+
 # Correct misassemblies for long reads using Tigmint
 tigmint-long: \
-	$(draft).$(reads).cut$(cut).molecule.size$(minsize).trim$(trim).window$(window).span$(span).breaktigs.fa \
+	$(draft).$(reads).cut$(cut).molecule.size$(minsize).dist$(dist).trim$(trim).window$(window).span$(span).breaktigs.fa \
```

Comment:
Since we're adding `dist` to the naming scheme for Tigmint-long, I think we should for Tigmint (with linked reads) too for consistency? Since that's a common parameter?